src/fieldValidation.ts

```diff
@@ -20,11 +32,18 @@ function isValidUrl(url: string): boolean {
 		type: 'string',
 		format: 'uri',
 	};
+
 	return ajv.validate(uriSchema, url);
 }
 
-function isValidPhoneNumber(phoneNumber: string): boolean {
-	return phone(phoneNumber).isValid;
+function isValidBoolean(bool: string): boolean {
+	const formattedBoolean = bool.toLowerCase();
```

Comment:
Also yeah just to respond here too: I think that we shouldn't do this formatting / do any kind of cleanup on the values.  If what they gave us doesn't pass validation according to AJV then it's not valid according to the json specification, and so I think we should consider it invalid and it becomes the burden of the client to give us proper booleans.

Reply:
Right this makes sense long term, though I do wonder about presenting data back to users, but I suppose that's an issue for when we start working with real users

Reply:
@hminsky2002 Early on there was a conversation about whether we are data curators or data scavengers. Ultimately it would be good to have a curator (and we'd often like to be it), but at first we need to be scavengers. We have to tread carefully about imposing hygiene or structure to the data without buy-in from the various prospective integrators and stakeholders.

Reply:
I hear ya BUT the data we are saving at this stage is the unchanged data, and saying that the unchanged data is valid when it isn't actually valid is a logical error.

To justin's point, eventually we may want to attempt to correct / massage the formatting of fields in certain cases (at which point we'd store both the raw data AND the fixed data), but that isn't what we're building here!